release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -250,6 +250,11 @@ For more information, see xref:../support/gathering-cluster-data.adoc#about-must
 
 Beginning with {product-title} {product-version}, you can upgrade or downgrade the firmware for a bare-metal node to a specific version. Metal^3^ provides the `HostFirmwareComponents` resource, which describes BIOS and baseboard management controller (BMC) firmware versions. Upgrading or downgrading firmware is useful when deploying an {product-title} cluster on bare metal with validated patterns that have been tested against specific firmware versions. See xref:../post_installation_configuration/bare-metal-configuration.adoc#about-the-hostfirmwarecomponents-resource_post-install-bare-metal-configuration[About the HostFirmwareComponents resource] for additional details.
 
+[id="ocp-4-16-editing-the-baremetalhost-resource_{context}"]
+==== Editing the BareMetalHost resource
```

Comment:
Editing a BMH (other fields) was always possible. What's special now is that the BMC address of a BMH can now be edited (if the BMH is Registering or by detaching it). Editing a BMC address after BMH creation was not possible earlier.

Reply:
Done.